Simplenote.xcodeproj/project.pbxproj

```diff
@@ -22,7 +22,7 @@
 			isa = PBXAggregateTarget;
 			buildConfigurationList = B5C7BF98230C59F5000DEC91 /* Build configuration list for PBXAggregateTarget "SimplenoteSecrets" */;
 			buildPhases = (
-				B5C7BF9E230C5A79000DEC91 /* ShellScript */,
+				B5C7BF9E230C5A79000DEC91 /* Copy Secrets */,
```

Comment:
I had forgotten about this, but we had this neat aggregate target that runs before the Simplenote target build and copies the `SPCredentials.swift` file into the path that the codebase expects.

What I did in this PR was to move the logic into a dedicated script (see above) to make it easier to work on and point the source file to `~/.configure/simplenote-ios/secrets/`